performance_manager/lib/postgres_schema.py

```diff
@@ -54,9 +23,6 @@ class VehiclePositionEvents(SqlBase):  # pylint: disable=too-few-public-methods
     start_time = sa.Column(sa.Integer, nullable=True)
     vehicle_id = sa.Column(sa.String(60), nullable=False)
     hash = sa.Column(sa.BigInteger, nullable=False, index=True, unique=False)
-    updated_on = sa.Column(
-        sa.DateTime, server_default=sa.func.now(), server_onupdate=sa.func.now()
-    )
```

Comment:
get rid of these, mypy doesn't like the declaration and I don't think they're functioning correctly

Reply:
is it the `on_update` that is causing problems?

Reply:
I think it's the `server_onupdate`. the normal way i'm used to doing `updated_on` columns involves triggers and not just with a field definition. so I think if we need an `updated_on` field we'll have to dig more into creating triggers with sqlalchemy